ecommerce/views.py

```diff
@@ -202,3 +205,27 @@ def get(self, request, username, *args, **kwargs):  # pylint: disable=unused-arg
         )
 
         return Response(serialize_user_bootcamp_runs(user=user))
+
+
+class CheckoutDataViewSet(ReadOnlyModelViewSet):
```

Comment:
I know our existing endpoint returns all bootcamps that a user can pay for, but I'm pretty positive that we're not sticking with that approach. Users will only need to get this checkout data for a specific application, and you can probably just return a 404 if no application id was provided